storage/bootstrap/bootstrapper/peers/source.go

```diff
@@ -320,7 +375,7 @@ func (s *peersSource) incrementalFlush(
 			segment, err := stream.Segment()
 			if err != nil {
 				tmpCtx.BlockingClose()
-				blockErr = err // Need to call prepared.Close, avoid return
+				blockErr = err // Need to  call prepared.Close, avoid return
```

Comment:
Any reason to add an extra space here?  (just want to avoid git blame being muddied if we don't have to touch this line)

Reply:
nah that was a mistake haha